admin/load.php

```diff
@@ -403,6 +403,41 @@ function perflab_admin_pointer( $hook_suffix ) {
 	$current_user = get_current_user_id();
 	$dismissed    = explode( ',', (string) get_user_meta( $current_user, 'dismissed_wp_pointers', true ) );
 
+	/*
+	 * Show an admin pointer if module is active to prompt
+	 * for an action to use the standalone plugin instead.
+	 */
+	$available_module_names = perflab_get_modules_compatible_with_standalone_plugin();
+	if (
+		! empty( $available_module_names )
+		&& current_user_can( 'install_plugins' )
+		&& current_user_can( 'activate_plugins' )
+		&& ! in_array( 'perflab-module-migration-pointer', $dismissed, true )
+	) {
+		// Enqueue the pointer logic and return early.
+		wp_enqueue_style( 'wp-pointer' );
+		wp_enqueue_script( 'wp-pointer' );
+		add_action(
+			'admin_print_footer_scripts',
+			static function () {
+				$content  = __( 'Your site is using modules which will be removed in the future in favor of their equivalent standalone plugins.', 'performance-lab' );
+				$content .= ' ' . sprintf(
```

Comment:
Possible to do this in one string instead of doing string concatenation?

Reply:
Updated in https://github.com/WordPress/performance/pull/910/commits/c8ac7e0698bf0a3cfb53d6585b46fa1c2d3b437b